geopandas/tests/test_plotting.py

```diff
@@ -321,14 +327,14 @@ def setup_method(self):
         self.series = GeoSeries([poly, line, point])
         self.df = GeoDataFrame({'geometry': self.series, 'values': [1, 2, 3]})
 
+    @pytest.mark.xfail
```

Comment:
Yes, I couldn't get it passing, but now looked more into it, and it was actually still a real bug for mixed geometries! Fixed now